README.md

```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```

Comment:
- [ ] In order to get into the frontend app, the user needs to at least type their username.  That way, different users can test the app and have their own reservations/bookings. 👍